lib/s3-service.js

```diff
@@ -11,7 +11,7 @@ const { getSignedUrl } = require('@aws-sdk/s3-request-presigner');
 const { streamToBuffer } = require('./utils');
 
 // Export
-exports.client = new S3Client();
+exports.setClient = (config) => (exports.client = new S3Client(config));
```

Comment:
I'd store the client in a variable with `new S3Client()` as the default,
and expose a setConfig or overrideConfig function that sets the S3 configuration.

Wdyt?